client/web/src/repo/constants.ts

```diff
@@ -2,3 +2,13 @@ export const LogsPageTabs = {
     COMMANDS: 0,
     SYNCLOGS: 1,
 } as const
+
+export enum CodeHostTypes {
```

Comment:
nit: the type only represents a single code host type, so I think it makes sense for this to be singular

```suggestion
export enum CodeHostType {
```